samples/run.ts

```diff
@@ -17,7 +20,57 @@ if (selection) {
     samples = [selection];
 }
 
+async function createProject(projectId: string, coreApiObject: coreApi.ICoreApi) : Promise<void> {
```

Comment:
I don't think we need to pass in the core api? Can probably declare at the file level so we can easily use it anywhere? Since this is just a test file we don't need to be as strict.